src/ast/stmt.rs

```diff
@@ -36,16 +36,24 @@ pub struct AssignStmt {
 
 #[derive(Debug, Eq, PartialEq, Serialize, Clone, new)]
 pub struct DeclStmt {
-    /// The type of the declared variable(s).
+    /// The declared variable(s).
+    pub variables: Vec<VarDecl>,
+    /// The expression(s) being assigned to the declared variables.
+    pub expressions: Vec<Expr>,
```

Comment:
We should probably specify in the doc comment the convention: one expression to many variables indicates a tuple, equal lengths indicate multiple declarations.